s3transfer/futures.py

```diff
@@ -17,6 +17,8 @@
 from collections import namedtuple
 from concurrent import futures
 
+from botocore.context import get_context
```

Comment:
The current version requirement for s3transfer is:

```python
requires = [
    'botocore>=1.36.0,<2.0a.0',
]
```

We should bump the minimum supported version to prevent import errors since the `botocore.context` won't exist in versions between 1.36.0 and the version that releases https://github.com/boto/botocore/pull/3389